macros/REST_Axion_XMMAngleEffCSV.C

```diff
@@ -0,0 +1,131 @@
+#include "TCanvas.h"
+#include "TLatex.h"
+#include "TLegend.h"
+#include "TRestTask.h"
+#include "TRestTools.h"
+
+#ifndef RestTask_Axion_XMMAngleEffCSV
+#define RestTask_Axion_XMMAngleEffCSV
+
+//*******************************************************************************************************
+//*** Description: It creates an efficiency curve from different runs under different angles. 
+//*** It's also possible to compare data from a csv file with this curve.
+//*** 
+//***
+//*** --------------
+//*** Usage: restManager XMMAngleEffCSV [pathAndPattern = "/trueWolter/OpticsBench_Yaw_*_Dev_*_BabyIAXO_Run*.root", csvFile = ""]
```

Comment:
We will usually give the first argument as a mandatory input, the second argument is optional, and thus it is defined between square brackets [ ].

```suggestion
//*** Usage: restManager XMMAngleEffCSV pathAndPattern [csvFile]
```

Reply:
I still don't understand what exactly I then have to put into the command line to put in a different csv file. Do I have to put the brackets there then?

Reply:
No, square brackets are not necessary, are simply a documentation format that tells the user that this argument is optional. It is commonly used in UNIX commands, for example, see `man ls` --> 

```
ls [-@ABCFGHILOPRSTUWabcdefghiklmnopqrstuvwxy1%,] [--color=when] [-D format] [file ...]
```

Reply:
If you agree, then just commit the suggestion, the pipeline is green again